content/graphql/quick-start.md

```diff
@@ -166,6 +166,8 @@ GraphQLModule.forRoot({
 
 The `autoSchemaFile` indicates a path where your automatically generated schema will be created. Additionally, you can pass the `buildSchemaOptions` property - an options object which will be passed in to the `buildSchema()` function (from the `type-graphql` package).
 
+Please keep in mind that you have to create a query to start generating a schema, a mutation or subscription will not work.
```

Comment:
```suggestion
> info **Hint** A query must be created before schema generation takes place. A mutation or subscription will otherwise not work as expected.
```